src/modules/template/service.test.js

```diff
@@ -0,0 +1,95 @@
+const { Mongol } = require('@albert-team/mongol')
+const TemplateService = require('./service')
+
+// MONGO_URL is just an ENV variable set by @shelf/jest-mongodb, not the actual one used in production
+const { MONGO_URL } = process.env
+
+describe('test suite for TemplateService', () => {
+  const mongol = new Mongol(MONGO_URL, 'testdb')
+  let db
+  let service
+
+  beforeAll(async () => {
+    await mongol.connect()
+    db = mongol.database
+  })
+
+  beforeEach(async () => {
+    await db.dropDatabase()
+    service = new TemplateService(mongol)
+  })
+
+  afterAll(async () => {
+    await mongol.disconnect()
+  })
+
+  describe('test suite for TemplateService.create()', () => {
+    test('Create Template', async () => {
+      const mockTemplate = {
+        name: 'test1',
+        urlPattern: 'http://*.com',
+        version: 1,
+        targets: [
+          {
+            name: 'name1',
+            cssSelector: 'h1',
+            type: 'string'
+          },
+          {
+            name: 'name2',
+            cssSelector: 'h2',
+            type: 'string'
+          }
+        ]
+      }
+      const template = await service.create(mockTemplate)
+      expect(template._id).toBeDefined()
+      expect(template).toMatchObject(mockTemplate)
+    })
+  })
+
+  describe('test suite for TemplateService.get()', () => {
+    const mockTemplate1 = {
+      name: 'test1',
+      urlPattern: 'http://*.com',
+      version: 1,
+      targets: [
+        {
+          name: 'name1',
+          cssSelector: 'h1',
+          type: 'string'
+        },
+        {
+          name: 'name2',
+          cssSelector: 'h2',
+          type: 'string'
+        }
+      ]
+    }
+
+    const mockTemplate2 = {
+      name: 'test2',
+      urlPattern: 'http://*.net',
+      version: 1,
+      targets: [
+        {
+          name: 'name1',
+          cssSelector: 'h1',
+          type: 'string'
+        }
+      ]
+    }
+    test('Get All Templates', async () => {
```

Comment:
```suggestion
    test('get all templates', async () => {
```
T để ý ông hay viết in hoa từng từ như này, nhưng theo convention thì chỉ cần viết thường thôi, trừ khi nó là tên class, function không thể viết khác đi được.